app/experimenter/experiments/forms.py

```diff
@@ -1271,10 +1272,42 @@ class NormandyIdForm(ChangeLogMixin, forms.ModelForm):
     normandy_id = forms.IntegerField(
         label="Recipe ID",
         widget=forms.TextInput(
-            attrs={"class": "form-control", "placeholder": "Recipe ID"}
+            attrs={"class": "form-control", "placeholder": "Main Recipe ID"}
         ),
     )
 
+    other_normandy_ids = forms.CharField(
+        label="Other Recipe IDs",
+        widget=forms.TextInput(
+            attrs={
+                "class": "form-control",
+                "placeholder": "Other Recipe IDs (Optional)",
+            }
+        ),
+        required=False,
+    )
+
+    def clean_other_normandy_ids(self):
+        other_ids_list = []
+
+        if self.cleaned_data["other_normandy_ids"]:
+            ids = re.split(",", self.cleaned_data["other_normandy_ids"])
+            for id in ids:
+                if re.match(r"^[0-9]+$", id.strip()):
+                    if "normandy_id" in self.cleaned_data:
+                        if int(id) == self.cleaned_data["normandy_id"]:
+                            raise forms.ValidationError(
+                                "Duplicate IDs are not accepted."
+                            )
+                        else:
+                            other_ids_list.append(int(id))
+                else:
+                    raise forms.ValidationError(
```

Comment:
Instead of doing this with a bunch of `else` statements that get deeper and deeper, you could flatten it more like

    if not some_check:
        raise forms.ValidationError('🐧')
    if not some_other_check:
        raise forms.ValidationError('🐻')

Reply:
did some flattening..